rhel/SPECS/dbaas-tools.spec

```diff
@@ -44,15 +56,27 @@ cd src/github.com/kubernetes-sigs/aws-iam-authenticator-%{commit_aws}
 sed -i '/dockers:/,+35d' .goreleaser.yaml
 make build
 
+cd %{_builddir}/kubernetes-%{commit_k8s}/
+export GOPATH="$(pwd)"
+export KUBE_GIT_TREE_STATE="clean"
+export KUBE_GIT_COMMIT=%{commit_k8s}
+export KUBE_GIT_VERSION=%{version_k8s}
+export KUBE_EXTRA_GOPATH=$(pwd)/Godeps/_workspace
```

Comment:
https://github.com/kubernetes/kubernetes/tree/v1.16.8/Godeps (everything is in `vendor/` now) That line should be removed

Reply:
Gotcha, will remove

Reply:
Done in: f85fd3e